src/main/java/org/cryptomator/ui/preferences/UpdatesPreferencesController.java

```diff
@@ -70,18 +116,74 @@ public ReadOnlyStringProperty latestVersionProperty() {
 	}
 
 	public String getLatestVersion() {
-		return latestVersion.get();
+		return latestVersion.isNotNull().get() ? latestVersion.get() : "-";
 	}
 
 	public String getCurrentVersion() {
 		return currentVersion;
 	}
 
+	public ObservableValue<Instant> lastSuccessfulUpdateCheckProperty() {
+		return lastSuccessfulUpdateCheck;
+	}
+
+	public String getLastSuccessfulUpdateCheck() {
+		Instant lastCheck = lastSuccessfulUpdateCheck.getValue();
+		if (lastCheck != null && !lastCheck.equals(Settings.DEFAULT_LAST_SUCCESSFUL_UPDATE_CHECK)) {
```

Comment:
Do we have to compare to the settings? we have the updateCheckerState and can check if it is "never"

Reply:
Yes, at application start, the `updateCheckerState` is set to `NOT_CHECKED` until an update check is performed. Therefore, comparing with the state might not be appropriate.